http/templates/base.html.tera

```diff
@@ -2,40 +2,36 @@
 {% block templates %}
 {% endblock %}
 <html lang="en">
-	<head>
```

Comment:
There seemed to be a mix of tab and space indenting here and the rest of the repo seemed to use spaces so I changed everything to spaces